src/amphtml-email/20_Samples/Bookmark_Article_Email.html

```diff
@@ -0,0 +1,123 @@
+<!---{
+ "preview": "amp4email",
+ "skipValidation": "true",
+ "draft": "true"
+}--->
+<!--
+## Introduction
+
+This sample showcases how to use AMPHTML email to create an email with a
```

Comment:
Can you give a more abstract overview of what this sample demonstrates, e.g.: "This sample demonstrates how to let users perform an action (maybe transaction) in an AMPHTML email. "